compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -727,6 +679,7 @@ generateSrcPkgFromLf getUnitId mbSdkPrefix pkg = do
         ["{-# LANGUAGE NoDamlSyntax #-}"
         , "{-# LANGUAGE NoImplicitPrelude #-}"
         , "{-# LANGUAGE TypeOperators #-}"
+        , "{-# OPTIONS_GHC -Wno-unused-imports #-}"
```

Comment:
it doesn’t really make sense to warn about unused imports in autogenerated code and it makes the build very noisy.